File: Backend/CliqueWebService/Helpers/Models/Currency.cs
using System.Text.Json.Serialization;

namespace CliqueWebService.Helpers.Models
{
    public class Currency
    {
        public int currency_id { get; set; }
        public string currency_name { get; set; }
        public string currency_abbr { get; set; }
    }
}

File: Client/Clique/Common/Models/Currency.swift
//
//  Currency.swift
//  Clique
//
//  Created by Infinum on 28.11.2022..
//

import Foundation

struct Currency: Codable {
    let id: Int
    let name: String
    let abbreviation: String
    
    enum CodingKeys: String, CodingKey {
        case id = "currency_id"
        case name = "currency_name"
        case abbreviation = "currency_abbr"
    }
}
